### ISSUE TITLE
Angular 17

### ISSUE THREAD

**Issue by kurtisblaine** [alvarotrigo/angular-fullpage#130]
Greetings,

Latest angular not working with the module:
https://stackblitz.com/edit/angular-1cqwpw?file=src%2Fapp%2Fapp.module.ts

I appreciate the good work, I just bought the license & am eager to see this in action. 
Let me know if you need any more details.

**Comment by alvarotrigo** [comment]
@alidihaw any idea what this can be?

**Comment by alidihaw** [comment]
Should working with update from this pr: https://github.com/alvarotrigo/angular-fullpage/pull/134

**Comment by kurtisblaine** [comment]
Any additional information i can provide?

**Comment by alvarotrigo** [comment]
> Any additional information i can provide?

Yes, have you tried with https://github.com/alvarotrigo/angular-fullpage/pull/134?

**Comment by kurtisblaine** [comment]
Can you publish through npm so i can get the changes?

**Comment by kurtisblaine** [comment]
@alidihaw @alvarotrigo

**Comment by kurtisblaine** [comment]
👍

**Comment by kurtisblaine** [comment]
Otherwise provide a way i am supposed to consume the changes - for i am unsure.

**Comment by alvarotrigo** [comment]
@alidihaw I just merged your commit and added a couple of examples for Angular 18 and 19. 
If you want to review anything, I'd be happy to approve any other pull request! 👍
